sdk/go/common/util/cmdutil/spinner.go

```diff
@@ -104,21 +109,22 @@ func (spin *ttySpinner) Reset() {
 // dotSpinner is the spinner that can be used when standard out is not a tty. In this case, we just write a single
 // dot on each tick.
 type dotSpinner struct {
+	color      colors.Colorization
 	prefix     string
 	hasWritten bool
 }
 
 func (spin *dotSpinner) Tick() {
 	if !spin.hasWritten {
-		fmt.Print(spin.prefix)
+		fmt.Print(spin.color.Colorize(colors.Yellow + spin.prefix))
```

Comment:
@susanev suggested it to help contrast from resource creations in white.

Reply:
I should probably add a reset here and then manually color it inside the printf down here just in case things get interrupted and we mess up a terminal